jobs/rootfs-builder.jpl

```diff
@@ -106,10 +109,10 @@ node("debos && docker") {
     }
 
     j.dockerPullWithRetry(docker_image).
-        inside(" --privileged --device /dev/kvm ") {
+        inside(" --privileged --device /dev/kvm -v /dev:/dev") {
 
         stage("Build") {
-            timeout(time: 8, unit: 'HOURS') {
+            timeout(time: 12, unit: 'HOURS') {
```

Comment:
Maybe these 2 changes should be split into 2 commits. The we may merge the timeout one into `main` but not the `/dev/` one, or make the timeout a parameter.

Reply:
Ok, will do

Reply:
Done